src/test/companyname/OrgTreeValidatorTest.java

```diff
@@ -0,0 +1,260 @@
+package test.companyname;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+
+import java.util.List;
+
+
+
+
+
+
+
+import org.junit.Test;
+
+import com.companyname.model.Company;
+import com.companyname.model.Department;
+import com.companyname.model.OrgTreeItem;
+import com.companyname.model.OrgTreeType;
+import com.companyname.model.TreeNode;
+import com.companyname.model.User;
+import com.companyname.util.CompanyGenerator;
+import com.companyname.util.DepartmentGenerator;
+import com.companyname.util.UserGenerator;
+
+public class OrgTreeValidatorTest {
+
+	private OrgTreeValidator orgTreeValidator;
+	
+	private Company company;
+	
+	private Department department1;
+	private Department department2;
+	
+	private User user1;
+	private User user2;
+	
+	@BeforeClass
+	public void setup() {
+		orgTreeValidator = new OrgTreeValidator();
+		
+		company = CompanyGenerator.getRandomCompany();
+		
+		department1 = DepartmentGenerator.getRandomDepartment(company);
+		department2 = DepartmentGenerator.getRandomDepartment(company);
+		
+		user1 = UserGenerator.getRandomUser(company, department1);
+		user2 = UserGenerator.getRandomUser(company, department2);
+	}
+		
+	
+//	@Test
+//	public void testOrgChartValid() {
+//		//For these random Users identify the distinct companies and then create a valid org chart for each
+//		List<User> userList = UserGenerator.getRandomListOfUsers(3000);
+//		
+//		TreeNode<OrgTreeItem> orgTree;
+//		
+//		
+//		
+//		Assert.fail("Not yet implemented");
+//	}
+	
+	@Test
+	/*
+	 * Valid Tree
```

Comment:
Made the trees a little more readable, even though the tab/spaces are causing this to look like garbage in this PR. I'll update it to spaces and push another commit.